src/style.css

```diff
@@ -1 +1,99 @@
 /* add styles here */
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-weight: 300;
+
+  --placeholder-font-color: #3d4e61;
+}
+
+@font-face {
```

Comment:
Generally we put font-faces at the start of our file or even on other files, but this is just a minor thing